assets/indexCtrl.js

```diff
@@ -114,7 +107,7 @@ module.exports = function ($scope, io) {
         if (data && data.err) {
           $scope.addRefError = data.err;
         } else {
-          $scope.addRefError = "Already added that URL.";
+          $scope.addRefError = "There was an error.";
```

Comment:
The server actually sends 'Already added that URL' in `data.err` if the URL was already added, so this just prevents the misleading messages.